core/sql/udrserv/udrserv.cpp

```diff
@@ -230,35 +230,6 @@ void InitializeJavaOptionsFromEnvironment(LmJavaOptions &javaOptions
 {
    char *val;
 
-   if ((val = getenv("CLASSPATH")))
-   {
-      javaOptions.addSystemProperty("java.class.path", val);
-   }
-   char maxHeapOption[64];
-   int maxHeapEnvvarMB = DEFAULT_JVM_MAX_HEAP_SIZE;
```

Comment:
You can remove all the defines related to this code at the top of this file. For some reason, I was not able to comment at that line.